website/docs/guides/getting_started.html.markdown

```diff
@@ -136,76 +136,51 @@ New Relic alerts are great, but they're even better when combined with good noti
-resource "newrelic_alert_policy_channel" "alert_policy_email" {
-  policy_id  = newrelic_alert_policy.alert_policy_name.id
-  channel_ids = [
-    newrelic_alert_channel.alert_notification_email.id
-  ]
-}
-```
-
-This example will send an email to the specified recipients whenever the associated alert condition is triggered. If you would like to send notifications via different modalities, such as Slack, the [alert channel](/providers/newrelic/newrelic/latest/docs/resources/alert_channel) resource supports multiple types of channels. Refer to the [additional alert channel examples](/providers/newrelic/newrelic/latest/docs/resources/alert_channel#type) for more examples in configuring your different notification channels.
-
-## A Note About Secrets
-
-As part of a `newrelic` resource, there is often some amount of configuration
-that is required in order for a resource to reach its full potential.  In some
-cases, once a given entity is created, API results will obscure the values for
-items that are deemed to be secret.  As a result, Terraform is unable to make
-an accurate detection of a resource state, and so marks a resource as changed
-for every run.
-
-Consider the following example.
-
-```hcl
-resource "newrelic_alert_channel" "slack" {
-  name = "slack"
-  type = "slack"
-
-  config {
-    channel = "test"
-    url     = "https://hooks.slack.com/services/xxxx/xxxxx"
+resource "newrelic_notification_channel" "sample_notification_channel" {
+  name           = "Sample Notification Channel"
+  type           = "EMAIL"
+  destination_id = newrelic_notification_destination.sample_notification_destination.id
+  product        = "IINT"
+  
+  property {
+    key   = "subject"
+    value = "Sample Email Subject"
   }
 }
-```
-
-The resource above yields the following Terraform plan.
-
-    -/+ newrelic_alert_channel.slack (new resource required)
-          id:                    "2344397" => <computed> (forces new resource)
-          config.%:              "1" => "2" (forces new resource)
-          config.channel:        <sensitive> => <sensitive> (attribute changed)
-          config.url:            <sensitive> => <sensitive> (forces new resource)
-          name:                  "slack" => "slack"
-          type:                  "slack" => "slack"
 
-To avoid the resource being marked as changed every run, the following can be
-implemented for the resource.
-
-```hcl
-resource "newrelic_alert_channel" "slack" {
-  ...
-  lifecycle {
-    ignore_changes = ["config"]
+resource "newrelic_workflow" "sample_workflow" {
+  name                  = "Sample Workflow"
+  muting_rules_handling = "NOTIFY_ALL_ISSUES"
+  
+  issues_filter {
+    name = "Issue Filter"
+    type = "FILTER"
+    predicate {
+      attribute = "labels.policyIds"
+      operator  = "EXACTLY_MATCHES"
+      values    = [newrelic_alert_policy.alert_policy_name.id]
+    }
+  }
+  
+  destination {
+    channel_id = newrelic_notification_channel.sample_notification_channel.id
```

Comment:
A note for @kidk - based on your feedback, I did want to remove `issues_filter`, but I think this is necessary to link the workflow to the alert policy, to keep it similar to the older example in which an `alert_channel` is linked to the alert policy previously created.

Please let me know your thoughts